examples/ChatQnA/deploy/AIPC.rst

```diff
@@ -0,0 +1,7 @@
+.. _ChatQnA_deploy_aiPC:
+
+
+Single Node On-Prem Deployment: AI PC
+#####################################
```

Comment:
Do we use markdown or rst format?

Reply:
In general you can use either reST or markdown for content.